app/actions/user/users_test.go

```diff
@@ -0,0 +1,201 @@
+package user_test
+
+import (
+	"mjm/app"
+	"mjm/app/models"
+	"net/http"
+	"testing"
+
+	"github.com/gobuffalo/suite/v4"
+)
+
+type ActionSuite struct {
+	*suite.Action
+}
+
+func Test_ActionSuite(t *testing.T) {
+	as := &ActionSuite{suite.NewAction(app.New())}
+	suite.Run(t, as)
+}
+
+func (as *ActionSuite) Test_List() {
+
+	department := models.Department{
+		Name:        "name ",
+		Description: "description",
+	}
+
+	as.NoError(as.DB.Create(&department))
+
+	user := models.User{
+		FirstName:    "Joaquin",
+		LastName:     "Olivo",
+		PhoneNumber:  "3042015706",
+		DepartmentID: department.ID,
+	}
+
+	as.NoError(as.DB.Create(&user))
+
+	res := as.HTML("/users").Get()
+	body := res.Body.String()
+	as.Contains(body, user.FirstName)
+
+}
+
+func (as *ActionSuite) Test_Create() {
+
+	department := models.Department{
+		Name:        "name ",
+		Description: "description",
+	}
+
+	as.NoError(as.DB.Create(&department))
+
+	user := models.User{
+		FirstName:    "Joaquin",
+		LastName:     "Olivo",
+		PhoneNumber:  "3042015706",
+		DepartmentID: department.ID,
+	}
+
+	res := as.HTML("/create-user").Post(user)
+
+	as.Equal(res.Code, http.StatusSeeOther)
+	as.Equal("/users/", res.Location())
+	userArray := []models.User{}
+	as.DB.All(userArray)
+	for _, v := range userArray {
+		as.Equal(v.FirstName, user.FirstName)
+	}
```

Comment:
Please add a validation of the length of "userArray", and I suggest you rename the variable. Just "users" would be ok.